src/backend/distributed/commands/utility_hook.c

```diff
@@ -150,10 +149,13 @@ multi_ProcessUtility(PlannedStmt *pstmt,
 		return;
 	}
 
-	checkExtensionVersion = IsCitusExtensionStmt(parsetree);
-	if (EnableVersionChecks && checkExtensionVersion)
+	if (IsCitusExtensionStmt(parsetree))
 	{
-		ErrorIfUnstableCreateOrAlterExtensionStmt(parsetree);
+		/* we intercept the creation of the citus extension, this allows us to do some checks and enable ssl if disabled */
```

Comment:
comment is out dated, actually all changes here are non-functional now after the refactor, will revert